nextflow_schema.json

```diff
@@ -139,51 +139,59 @@
             "fa_icon": "fas fa-cut",
             "properties": {
                 "trim_fastq": {
-                    "type": "string",
+                    "type": "boolean",
                     "fa_icon": "fas fa-cut",
-                    "description": "Run Trim Galore"
+                    "description": "Run Trim Galore",
+                    "hidden": true
                 },
                 "clip_r1": {
                     "type": "integer",
                     "fa_icon": "fas fa-cut",
                     "description": "Remove bp from the 5' end of read 1",
-                    "help_text": "With Trim Galore"
+                    "help_text": "With Trim Galore",
```

Comment:
not a very helpful text, in my opinion. Would easily also fit into the description.